questions/smallest-sum.md

```diff
@@ -0,0 +1,19 @@
+# Smallest Sum
+
+## Question
+
+Here is the problem statement:
+
+Given a list of integers, you will need to find sum that can be created for a given list. For example, if the list was [1,6,9,4,3], the smallest sum of the list would be 4(3+1). Below is a way that you could go about finding smallest sum in an array:
```

Comment:
cool question! Maybe try making the problem more clear. It could maybe be confusing in the first sentence where it says "you will need to find sum that can be created for a given list." Later in the sentence it is mentioned about smallest sum, maybe try adding that near the beginning to make idea more clear :)